static/js/util/api.js

```diff
@@ -128,3 +128,14 @@ export function checkout(courseId: string): Promise<CheckoutResponse> {
     })
   });
 }
+
+export function sendMail() {
```

Comment:
not used anywhere as part of this PR. i added this as a stub to make the actual integration easier. the actual issue (which i'm working on now): https://github.com/mitodl/micromasters/issues/889